### ISSUE TITLE
Kibana 5.6.10 support 

### ISSUE THREAD

**Issue by IdanWo** [ppadovani/KibanaNestedSupportPlugin#88]
Hey, 

We're using ElasticSearch 5.6.10. Will you plan to support it officially? 
As a workaround, I'm trying to install kibana 5.6.9 instead of kibana 5.6.10.

**Comment by ppadovani** [comment]
I'll port and test this weekend, look for a release Sunday or Monday morning.

**Comment by IdanWo** [comment]
An Update: 
-  Workaround works. Kibana 5.6.9 supports ElasticSearch 5.6.10 (a warning message is shown, though). We don't want to support 2 version of Kibana, so 5.6.10 release would be great. Thank you!
- I did noticed that document view called ‘Structured’ WASN'T added to the discover application. I don't know if it has any correlation with the versions conflict.

**Comment by ppadovani** [comment]
Ports completed:

5.6.10
5.6.11
6.3.2

**Comment by IdanWo** [comment]
Upgraded to 5.6.10 today, worked like a charm

**Comment by mxcoder** [comment]
Hi @ppadovani , your plugin looks really promising to what I'm trying to do, any plans on giving support for 6.4.0 yet? As my project is new I chose to use the latest ELK version.

I just tried the last version of your plugin, modified the package.json settings and give it a go but Kibana broken while trying to "optimize" the assets of your plugin. Let me know if there's anything I could do to help you, I'm very new to ELK but I'm familiar with Web stuff.

Thanks.
